CMakeLists.txt

```
@@ -131,7 +131,7 @@ endif()
 
 if (COMPILER_FAMILY_IS_MSVC)
   if (NOT BUILD_SHARED_LIBS)
-    target_compile_definitions(camp PUBLIC WIN_STATIC_BUILD)
+    target_compile_definitions(camp PUBLIC CAMP_WIN_STATIC_BUILD)
```

That looks great, and it seems like it works, one tweak to clean up something right here and I think this one is good to go.

```suggestion
    target_compile_definitions(camp PRIVATE CAMP_WIN_STATIC_BUILD)
```